sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -183,9 +184,9 @@ def _get_pred_args_from_metric(scitype, metric):
     return {}
 
 
-def _evaluate_window(x, meta):
+def _evaluate_window(iteration, x, meta):
```

Comment:
I think this change will break integration with `parallelize`.

Why is this change necessary? It does not seem to be?